ec2-cluster/variables.tf

```diff
@@ -0,0 +1,83 @@
+variable "aws_region" {
+  type    = string
+  default = "us-west-1"
+}
+
+variable "env" {
+  type    = string
+  default = "dev"
+}
+
+variable "instance_count" {
```

Comment:
This should be derived automatically based on the number of keys provided in the `instance_keys`.

Reply:
fixed https://github.com/Constellation-Labs/infrastructure_utils/pull/21/files